Documentation/Tutorials/edge-send-event-tutorial.md

```diff
@@ -0,0 +1,668 @@
+
+1. Make sure that the base URL for the Assurance session is set to the correct value (`edgetutorialapp://`), and try recopying the link and submitting again.
+   - For instructions on how to change the base URL value, see the section **Details on connecting to Assurance** under [Connect to the app](#2-connect-to-the-app)
+2. Make sure that the tutorial app is installed on the device. If it was already installed, try uninstalling it and reinstalling it.
+
+If in the app, after entering the PIN code and tapping **Connect**, you see the error: "Invalid Mobile SDK Configuration The Experience Cloud organization identifier is unavailable. Ensure SDK configuration is setup correctly. See documentation for more detail."
+
+1. Make sure that the mobile property used has Assurance installed, and that the property has been properly published.
+2. Make sure that the mobile property ID is set in the `ENVIRONMENT_FILE_ID` variable in `AppDelegate.swift`, then rebuild the app.
+
+</p></details>
+
+
+<details>
+  <summary> Connecting using QR code </summary><p>
+
+To connect using QR code:
+Prerequisites (see [Set up the Assurance session](#1-set-up-the-assurance-session) for details on QR code requirements):
+- Running app on a **physical device** with camera that can scan QR codes
+- App URL for deep linking is configured
+- App code for receiving link and connecting to Assurance is implemented
+
+1. Use physical device's camera to scan the QR code, which when tapped, should trigger a confirmation dialog to open the app.
+2. App should open and show the Assurance PIN screen to authenticate the session connection; enter the PIN from the session details and tap **Connect**
+
+</p></details>
+
+Once connected to Assurance, in the tutorial app, an Adobe Experience Platform icon (**1**) will appear in the top right corner of the screen with a green dot indicating a connected session.  
+<img src="../Assets/edge-send-event-tutorial/ios-assurance-connection.png" alt="Assurance Session Start - Web UI after connection" width="400"/>  
+
+In the web-based Assurance session, there is also an indicator in the top right that shows the number of connected sessions (which in this case should now show a green dot with "1 Client Connected" (**1**)).  
+<img src="../Assets/edge-send-event-tutorial/assurance-session-start.jpg" alt="Assurance Session Start - Web UI after connection" width="1100"/>  
+
+Notice how in the Assurance session Events view (**2**), there are already events populating as a consequence of the connection of the mobile app to the Assurance session (**3**); the Assurance extension itself emits events about the session connection and subsequently captures these events to display in the web-based session viewer. You can expect Assurance to capture all events processed by the AEP SDK from all other extensions as well.  
+
+### 3. Assurance Event transactions view - check for Edge events  
```

Comment:
This section now covers Events view and Event transactions view. Should we rename it simply  "Inspect events with Assurance" ? (same in table of content)